### DISCUSSION TITLE
Current yearly bill $9.84 why ?

### DISCUSSION THREAD

**Discussion by learning2016** [community/community#113128] [New to GitHub]
Hi
We changed to free plans on January 30th
But open the web page---billing and plans --》billing summary
Current yearly bill
$9.84
Next payment due
March 26, 2024

Is it possible to waive this part of the fee ？
Andy

**Comment by Akash1134** [comment]
Hi @learning2016 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by learning2016** [reply]
@Akash1134 
Thank you!
I sent the case five days ago, but I didn't get any response from the support staff.
![WX20240319-171057@2x](https://github.com/community/community/assets/18454348/bbd8d4c6-df73-4a86-84b9-23e8a6ad174b)
